Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -105,14 +104,16 @@ is ready to operate:
 
 Note, in above helm configuration the ``kubeProxyReplacement`` has been set to
 ``strict`` mode. This means that the Cilium agent will bail out in case the
-underlying Linux kernel support is missing. Without explicitly specifying a
-``kubeProxyReplacement`` option, helm uses ``kubeProxyReplacement`` with ``probe``
-by default which would automatically disable a subset of the features to implement
-the kube-proxy replacement instead of bailing out if the kernel support is
-missing. This makes the assumption that Cilium's BPF kube-proxy replacement would
-co-exist with kube-proxy on the system to optimize Kubernetes services. Given
-we've used kubeadm to deploy a kube-proxy-free setup, the ``strict`` mode
-has been used instead.
+underlying Linux kernel support is missing.
+
+Without explicitly specifying a ``kubeProxyReplacement`` option, helm uses
+``kubeProxyReplacement`` with ``probe`` by default which would automatically
+disable a subset of the features to implement the kube-proxy replacement instead
+of bailing out if the kernel support is missing. This makes the assumption that
```

Comment:
I think the current way is a bit clearer since right before that we're talking about strict mode and `This means that the Cilium agent will bail out in case the underlying Linux kernel support is missing.`, so given the prior context I think it's more clear in the probe part to talk about disabling components when `[...] kernel support is missing [...]`.

Reply:
It reads to me that we enable something in the kernel, and that something implements the replacement.